base/src/main/kotlin/io/goooler/demoapp/base/network/BaseRetrofitHelper.kt

```diff
@@ -32,9 +32,9 @@ abstract class BaseRetrofitHelper {
     }
   }
 
-  fun <T> create(service: Class<T>): T = retrofit.create(service)
+  fun <T : Any> create(service: Class<T>): T = retrofit.create(service)
 
-  inline fun <reified T> create(): T = create(T::class.java)
+  inline fun <reified T : Any> create(): T = create(T::class.java)
```

Comment:
Public-API inline function cannot access non-public-API 'private final val retrofit: Retrofit defined in io.goooler.demoapp.base.network.BaseRetrofitHelper'